resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -536,17 +558,31 @@ private InetSocketAddress nextNameServerAddress() {
             Promise<AddressedEnvelope<? extends DnsResponse, InetSocketAddress>> promise) {
 
         return query0(checkNotNull(nameServerAddr, "nameServerAddr"),
-                      checkNotNull(question, "question"),
-                      checkNotNull(promise, "promise"));
+                checkNotNull(question, "question"), null, checkNotNull(promise, "promise"));
```

Comment:
Instead of passing `null`, it would probably be better to pass `Collections.emptyList()`